js/packages/components/chat/message/UserMessageComponents.tsx

```diff
@@ -8,12 +8,16 @@ import { Maybe, useClient } from '@berty-tech/store/hooks'
 import { useStyles } from '@berty-tech/styles'
 
 import { pbDateToNum, timeFormat } from '../../helpers'
+import { extensions } from './extensions'
 import { InteractionUserMessage, ParsedInteraction } from '@berty-tech/store/types.gen'
 import { WelshMessengerServiceClient } from '@berty-tech/grpc-bridge/welsh-clients.gen'
+import linkify from 'linkify-it'
 
 const READ_MORE_MESSAGE_LENGTH = 325
 const READ_MORE_SUBSTR_LENGTH = 300
 
+linkify.tlds(extensions)
```

Comment:
Can't we avoid maintaining a list of TLDs within the app? We will have to update it endlessly.

Can you also add `true` as the second parameter to keep TLDs already managed by the lib as seen in this example https://github.com/markdown-it/linkify-it#example-1 ? I think won't have to add ccTLDs/gTLDs in your list. Try sorting your list alphabetically, it might be easier to maintain.

Reply:
Yep 👍 
I'll use https://github.com/stephenmathieson/node-tlds to avoid maintaining this list 👍